dll/cpl/inetcpl/lang/de-DE.rc

```diff
@@ -122,14 +122,14 @@ STYLE WS_CAPTION | WS_CHILD | WS_DISABLED
 FONT 8, "MS Shell Dlg"
 CAPTION "Connections"
 BEGIN
-    GROUPBOX        "Automatic configuration", IDC_STATIC, 4, 4, 312, 56
-    AUTOCHECKBOX    "Use Web Proxy Auto-Discovery (WPAD)", IDC_USE_WPAD, 10, 14, 200, 14, BS_TOP
-    AUTOCHECKBOX    "Use Proxy Auto-Config (PAC) script", IDC_USE_PAC_SCRIPT, 10, 28, 200, 14, BS_TOP
-    LTEXT           "Address:", IDC_STATIC, 10, 42, 40, 14
+    GROUPBOX        "Automatische Konfiguration", IDC_STATIC, 4, 4, 312, 56
+    AUTOCHECKBOX    "Web Proxy Auto-Discovery (WPAD) benutzen", IDC_USE_WPAD, 10, 14, 200, 14, BS_TOP
+    AUTOCHECKBOX    "Proxy Auto-Config (PAC) Skript benutzen", IDC_USE_PAC_SCRIPT, 10, 28, 200, 14, BS_TOP
+    LTEXT           "Addresse:", IDC_STATIC, 10, 42, 40, 14
     EDITTEXT        IDC_EDIT_PAC_SCRIPT, 50, 42, 160, 14, WS_VISIBLE | ES_AUTOHSCROLL | WS_DISABLED
-    GROUPBOX        "Proxy server", IDC_STATIC, 4, 66, 312, 42
-    AUTOCHECKBOX    "Use a proxy server", IDC_USE_PROXY_SERVER, 10, 76, 200, 14, BS_TOP | BS_MULTILINE
-    LTEXT           "Address:", IDC_STATIC, 10, 90, 40, 14
+    GROUPBOX        "Proxyserver", IDC_STATIC, 4, 66, 312, 42
+    AUTOCHECKBOX    "Proxyserver benutzen", IDC_USE_PROXY_SERVER, 10, 76, 200, 14, BS_TOP | BS_MULTILINE
+    LTEXT           "Addresse:", IDC_STATIC, 10, 90, 40, 14
```

Comment:
Adresse is written with a single d in german.

Reply:
It may then be worth while to also fix these two occurrences of "Addresse":
https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rosapps/applications/sysutils/systeminfo/lang/de-DE.rc;hb=9bdeaca56e8e01473d9858a1b7ea4bbdbbc836cf#l72
https://git.reactos.org/?p=reactos.git;a=blob;f=base/applications/rapps/lang/de-DE.rc;hb=9bdeaca56e8e01473d9858a1b7ea4bbdbbc836cf#l200

Reply:
the 2nd one is fixed within #3997 but I don't really care about rosapps.